tracer/build/crank/README.md

```diff
@@ -0,0 +1,51 @@
+Crank
+=====
+
+The crank jobs are primarily designed to run as part of the CI, but it can be useful to run them from your local machine
+if you're trying to optimize something.
+
+The scripts `run.sh` and `run-appsec.sh` are design to be used at part of the CI.
+
+The script `./run-adhoc.sh` should be run from a Linux dd-trace-dotnot development environment (WSL will do nicely). It can be used
+to run a throughput test with the tracer and / or profiler attached.
+
+Before running the script is run the tracer and profiler, if desired, source must be built. Use the following commands:
+
+```
+./tracer/build.sh BuildTracerHome
+# if the profiler is desired
+./tracer/build.sh BuildProfilerHome
+./tracer/build.sh CompileNativeLoader
+./tracer/build.sh PublishNativeLoader
+```
+
+Once the source is built the script can used with the following command line:
+
+```
+./run-adhoc.sh appsec_baseline linux_adhoc true
+```
+
+or
+
+```
+./run-adhoc.sh appsec_baseline linux_profiler_adhoc true
+```
+
+1st parameter - name of the test to run
+2nd parameter - name of the OS profile to use
+3rd parameter - true use the appsec tests definitions file, false for tracer
+
+
+The following variables need to be defined in `/var/opt/crank/variables.yml` on your development machine and you **must be connected to Appgate** to access these IPs.
+
+```
+variables:
+  linuxIp: "172.29.209.246"
+  linuxEndpoint: "http://172.29.209.246:5985"
+  linuxProfilerPath: "/home/ubuntu/tracer"
+  controllerIp: "172.29.219.181"
+```
+
+The AWS servers with the private IPs 172.29.209.246 & 172.29.219.181 are tagged with the names `crank-agent-adhock-app` and `crank-agent-adhock-load` respectively.
+They are on the `vpc-d1aac1a8 (sandbox)`, which allows there private IPs to be accessed via Appgate, where as the AWS servers for CI jobs are not accessible
+via Appgate.
```

Comment:
I'm a bit loathe to be embedding these IP addresses and discussing VPCs here, especially as it could change, and security teams probably will get a bit twitchy about it. I get it's convenient, but it seems like the advice should be to connect to a crank machine and copy the file from there?

Especially given this can only be used by Datadog employees, and is very much tied to our infrastructure, I think we should add this to the "devops" document where we describe the crank machines in Confluence

Reply:
Yes, you're right I'll move it to confluence.

Reply:
removed via rebase / force push